test/Microsoft.AspNetCore.Mvc.Razor.ViewCompilation.FunctionalTests/ApplicationConsumingPrecompiledViews.cs

```diff
@@ -16,9 +18,15 @@ public ApplicationConsumingPrecompiledViews(ApplicationConsumingPrecompiledViews
 
         public ApplicationTestFixture Fixture { get; }
 
-        [Fact]
-        public async Task ConsumingClassLibrariesWithPrecompiledViewsWork()
+        public static TheoryData SupportedFlavorsTheoryData => RuntimeFlavors.SupportedFlavorsTheoryData;
```

Comment:
Nah, it's more maintenance if you need to change signatures in the future. Same reasoning applies to why we choose to use `var xyz = 123` over being explicit in the type name.